docs/examples/ui-provider-v1.md

```diff
@@ -0,0 +1,74 @@
+# UI Provider v1
+
+This example demonstrates usage of [UI Provider v1 API](../types/ui-provider-v1.md)
+
+**Note**: Linter fills the messages with `version` and `key` keys to make telling difference between them easy for UI providers.
```

Comment:
"them" should probably be expanded to something like:

"the source of the `Message`s easy for ..."